server/webhook_test.go

```diff
@@ -52,6 +52,14 @@ func (fakeWebhookHandler) HandleJobs(_ context.Context, _ *gitlabLib.JobEvent) (
 	return nil, nil
 }
 
+func (fakeWebhookHandler) HandleDeployment(_ context.Context, _ *gitlabLib.DeploymentEvent) ([]*webhook.HandleWebhook, error) {
```

Comment:
Are there are any unit tests for these?

Reply:
@wiggin77, Added them, Please have a look